webapp/channels/src/components/root/root.tsx

```diff
@@ -600,7 +561,6 @@ export default class Root extends React.PureComponent<Props, State> {
                         </div>
                         <Pluggable pluggableName='Global'/>
                         <AppBar/>
-                        <SidebarRightMenu/>
```

Comment:
I think this needs to be added again (sidebar_mobile_right_menu.tsx), or was this being moved somewhere else?
![CleanShot 2024-08-30 at 00 31 57](https://github.com/user-attachments/assets/cbf2db9e-dfd7-4bf7-b066-4cdb241df056)

Reply:
thanks for finding that out. reverted. I have moved it other place but i see that it broke the css so i opened another [ticket](https://mattermost.atlassian.net/browse/MM-60369) to tackle that for now.